cg/meta/report/templates/macros/data_analysis/data_analysis.html

```diff
@@ -0,0 +1,116 @@
+{% from "/macros/data_analysis/qc_metrics/balsamic_qc_metrics.html" import balsamic_qc_metrics %}
+{% from "/macros/data_analysis/qc_metrics/mip_dna_qc_metrics.html" import mip_dna_qc_metrics %}
+{% from "/macros/data_analysis/qc_metrics/rnafusion_qc_metrics.html" import rnafusion_qc_metrics %}
+{% from "/macros/data_analysis/limitations.html" import apptag_limitations %}
+
+<!-- Data analysis results -->
+{% macro data_analysis(case) %}
+  <div class="container mb-5">
+  <h3 class="mb-4">Analys</h3>
+    {{ workflow_analysis(case=case) }}
+    <h4 class="mt-4 mb-3">Kvalitetskontrollsmetrik</h4>
+    {% if "balsamic" in case.data_analysis.workflow %}
+      {{ balsamic_qc_metrics(samples=case.samples, analysis_type=case.data_analysis.type) }}
+    {% elif "mip-dna" == case.data_analysis.workflow %}
+      {{ mip_dna_qc_metrics(samples=case.samples) }}
+    {% elif "rnafusion" == case.data_analysis.workflow %}
+      {{ rnafusion_qc_metrics(samples=case.samples) }}
+    {% endif %}
+    {{ apptag_limitations(applications=case.applications) }}
+  </div>
+{% endmacro %}
+
+<!-- Workflow analysis -->
+{% macro workflow_analysis(case) %}
```

Comment:
```suggestion
{% macro workflow_analysis(case) %}
{% set workflow = case.data_analysis.worflow  %}
```
You coud set its value here and use it downstream